doc/akri_connector/connector-metadata-schema.json

```
@@ -480,10 +465,46 @@
                     "eventConfigurationSchema": {
                       "$ref": "http://json-schema.org/draft-07/schema#",
                       "description": "The JSON schema for the \"eventConfigurationSchema\" field on an event."
+                    },
+                    "fields": {
+                      "type": "object",
+                      "description" : "Describes if this connector expects these fields to be provided in the asset defintion when deploying the connector. Also describes the expected shape of these values if they are supported.",
```

```suggestion
                      "description" : "Describes if this connector expects these fields to be provided in the event definition when deploying the connector. Also describes the expected shape of these values if they are supported.",
```

Similar to above. Technically it is the asset definition but we might want to be more specific that these are fields on the event

---

Also applies for the below descriptions